src/parser.ts

```diff
@@ -226,11 +226,12 @@ export async function preprocessDryRunOutput(
   // about them anyway and also there will be a correspondent line in the dryrun with these variables expanded.
   // Don't remove lines with $ without paranthesis, there are valid compilation lines that would be ignored otherwise.
   preprocessTasks.push(function (): void {
-    regexp = /.*\$\(.*/gm;
-    preprocessedDryRunOutputStr = preprocessedDryRunOutputStr.replace(
-      regexp,
-      ""
-    );
+    preprocessedDryRunOutputStr = preprocessedDryRunOutputStr
```

Comment:
@SeasClouds  Have you tested that this performs the same and does the same things as the original code?

Reply:
I have done some tests and the results are consistent with using regular expressions.

Reply:
In that case, I think that this is safe to go in, we should keep an eye on it when we officially release in case there are regressions, but I'm okay to merge this, and have people experiment it in the pre-release version.